velox/exec/RowsStreamingWindowBuild.cpp

```diff
@@ -102,6 +102,14 @@ void RowsStreamingWindowBuild::noMoreInput() {
 
 std::shared_ptr<WindowPartition> RowsStreamingWindowBuild::nextPartition() {
   VELOX_CHECK(hasNextPartition());
+
```

Comment:
@JkSelf : So without this change you were accumulating all the partitions, when it should've been streaming behavior.

But doesn't seem like those partitions are holding onto any substantial memory, else we would've caught this before.

Please can you clarify the reason for this fix.